src/schema/v2/previewSavedSearch.ts

```diff
@@ -74,6 +74,8 @@ const previewSavedSearchArgs: GraphQLFieldConfigArgumentMap = {
   },
 }
 
+const SUPPORTED_ATTRIBUTIONS_CLASSES = ["unique", "limited edition"]
```

Comment:
**suggestion**: Maybe `ALLOWED_RARITY_SUGGESTIONS` might be more intention-revealing?

Reply:
makes sense. I will make the change